addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/YamahaReceiverCommunication.java

```diff
@@ -88,44 +88,42 @@ public Zone getZone() {
     public void updateDeviceInformation(YamahaReceiverState state) throws IOException {
         Document doc = postAndGetXmlResponse(
                 "<?xml version=\"1.0\" encoding=\"utf-8\"?><YAMAHA_AV cmd=\"GET\"><System><Config>GetParam</Config></System></YAMAHA_AV>");
-        if (doc != null) {
```

Comment:
The `postAndGetXmlResponse` can't return null it returns either an object or an exception.